addons/cni-cilium/cilium.yaml

```diff
@@ -138,7 +161,6 @@ data:
   install-no-conntrack-iptables-rules: "false"
 
   auto-direct-node-routes: "false"
-  enable-bandwidth-manager: "false"
```

Comment:
Do we need this? It's removed from the upstream manifest.

Reply:
Can be removed.